core/src/main/java/com/mapzen/android/graphics/MapzenMap.java

```diff
@@ -649,7 +643,8 @@ public void onMarkerPick(final MarkerPickResult markerPickResult, final float po
         mapView.post(new Runnable() {
           @Override public void run() {
             if (markerPickResult != null) {
-              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker()));
+              listener.onMarkerPick(new BitmapMarker(markerManager, markerPickResult.getMarker(),
```

Comment:
Also the `FeaturePickListener` is the one which handles picking of `Markers` (the one backed by `Tangram.MapData`)